stripe-sms-receipt/.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules/
```

Comment:
Do we need a duplicate `.gitignore` file here? Shouldn't this be covered by the toplevel one?

Reply:
Ohh, I didn't realise that `twilio serverless:init ...` sets up a very elaborate `.gitignore` file, that's great. Will revert those changes 👍